packages/react-server-test-pages/pages/root/aboveTheFold.js

```diff
@@ -0,0 +1,25 @@
+import {ReactServerAgent, RootContainer, RootElement} from "react-server"; // eslint-disable-line
+
+export default class RootWhenPage {
+	handleRoute(next) {
+		this.data = ReactServerAgent.get('/data/delay?ms=200');
+		return next();
+	}
+	getElements() {
+		return [
+			<RootContainer>
+				<div>One</div>
+			</RootContainer>,
+			<RootElement when={this.data}><div>Two</div></RootElement>,
+			<RootContainer>
+				<div>Three</div>
+				<RootElement when={this.data}><div>Four</div></RootElement>
+			</RootContainer>,
+			<div>Five</div>,
```

Comment:
Might be nice to have some text in the content here that says what the viewer should expect to see if things are working properly.